pdf/utils.go

```diff
@@ -160,6 +160,11 @@ func readerToWriter(r *unipdf.PdfReader, w *unipdf.PdfWriter, pages []int) error
 		return err
 	}
 
+	// Add optional properties
```

Comment:
In the future we should consider if there are other document properties that we want to preserve, such as outlines, certain flags and such.  But that should be placed in a separate issue.

Reply:
I agree. There might be more info to copy to the output files. For the outlines, we might have to add a couple of more methods to the high-level outlines struct. Things like merging and splitting of outlines, so that I don't have to do the logic in unicli, as it does not really belong here.